### PR TITLE
Adding dark room maps

### PR DESCRIPTION
**Pull request by MattEqualsCoder**
- Added images and maps that can be selected
- Added voice commands for displaying and hiding the maps
- Added some random tracker comments

### CODE REVIEW

**Comment by MattEqualsCoder** [comment]
My plan is to probably release a new build shortly after betus's stream tomorrow. My thought is if he continues these async race streams, it's probably best to avoid people being on different versions unless there's a major bug that needs to be fixed.

File: src/Randomizer.SMZ3.Tracking/tracker.json [hunk 1]
```diff
@@ -1855,6 +1898,7 @@
         "Hey chat.",
         "Hello Twitch chat.",
         "Hey Twitch chat.",
+        [ "Hey chat, smile if you're only here because of me.", 0.1 ],
```
Comment (Vivelin) [inline src/Randomizer.SMZ3.Tracking/tracker.json:1901]: I love this

**Review by Vivelin — APPROVED** [review]
Just curious, why specifically dark rooms and not just all dungeon maps? Too much work right now?

**Comment by MattEqualsCoder** [comment]
Well, it's two fold. The first is that, yeah, getting together all of the dungeon maps and coming up with a good way of presenting them won't be easy with showing how different floors connect and everything.

But also, I feel like the dungeon maps would be the entire dungeon so you can figure out your route through the dungeon at a glance, whereas this is so you can get a closer look at individual rooms so you can navigate them.

Basically, I think they serve different purposes with what I was thinking.

**Review by Vivelin — APPROVED** [review]
